runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/ServerFactory.java

```diff
@@ -72,6 +73,14 @@ public abstract Server allocatePortAndCreate(
   public abstract Server create(
       BindableService service, Endpoints.ApiServiceDescriptor serviceDescriptor) throws IOException;
 
+  /**
+   * Creates an instance of this server at the address specified by the given service descriptor and
+   * binded by multi services. Server applies {@link
```

Comment:
`binded by multi services` -> `bound to multiple services`

Reply:
Fixed in commit https://github.com/apache/beam/pull/7168/commits/d97bd464982d8aaed0fe05f28ffd40d55ec42e55